src/test/java/no/nav/brregstub/service/BrregServiceTest.java

```diff
@@ -0,0 +1,264 @@
+package no.nav.brregstub.service;
```

Comment:
Kunne man delt opp noen av testene i mindre tester og flere tester? Er lettere å se hva som gikk galt hvis en "liten" test feiler istede for en stor en